Sprint_Tests/sofia_zahoruiko.md

```diff
@@ -0,0 +1,18 @@
+1. He had a deep understanding of the layout and working with advanced technologies at the frontend.
+After a short break of several years and a deep depression, plus listening to heavy music, he has poorly understanding of the modern trends and drinks a lot.
+If he could handle depression after his wife and kid death, he wouldn't have to find a job with a relevant salary now.
+
+2. If he had a goal and adequate salary expectations, he would have already working on the remote in the USA or in Bali with the prospect of moving to the Canary Islands and getting Spanish citizenship. 
+His dream from childhood is marrying Spanish and retiring early and growing grapes on the Balearic coast. 
```

Comment:
`...he would have already worked remotely` - wrong verb form/ word usage
`prespective` - opportinities for growth
Since his childhood he's been dreaming about marrying a Spanish lady...